doc/tutorials/SIR.ipynb

```diff
@@ -557,7 +557,7 @@
    "outputs": [],
    "source": [
     "# Build numerical solution\n",
-    "I_opt = SIR_UK.solve(n_days-1, n_days).fetch()[:,2]\n",
+    "# I_opt = SIR_UK.solve(n_days-1, n_days).fetch()[:,2]\n",
```

Comment:
Why is this line commented out?

Reply:
I'll have a look

Reply:
Because it is previously defined 4-5 cells above.